packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -19,12 +19,13 @@ import { ProblemManager } from './problem-manager';
 import { ProblemMarker } from '../../common/problem-marker';
 import { ProblemTreeModel } from './problem-tree-model';
 import { MarkerInfoNode, MarkerNode } from '../marker-tree';
-import { TreeWidget, TreeProps, ContextMenuRenderer, TreeNode, NodeProps, TreeModel, SelectableTreeNode } from "@theia/core/lib/browser";
-import { h } from "@phosphor/virtualdom/lib";
+import { TreeProps, ContextMenuRenderer, TreeNode, NodeProps, TreeModel, SelectableTreeNode } from "@theia/core/lib/browser";
 import { DiagnosticSeverity } from 'vscode-languageserver-types';
 import { Message } from '@phosphor/messaging';
 import URI from '@theia/core/lib/common/uri';
 import { UriSelection } from '@theia/core/lib/common/selection';
+import { TreeWidget } from '@theia/core/lib/browser/tree/tree-widget';
```

Comment:
Same as above. Cannot you leave the imports as they were? Perhaps I am overlooking something.

Reply:
Renamed to tree-react-widget before. This new import was added automatically by the IDE. I'm rolling back everything.